packages/vitest/src/integrations/chai/jest-expect.ts

```diff
@@ -129,9 +129,44 @@ export const JestChaiExpect: ChaiPlugin = (chai, utils) => {
   })
   def('toBe', function (expected) {
     const actual = this._obj
+    const pass = Object.is(actual, expected)
+
+    let toBeMessage = 'expected #{this} to be #{exp} // Object.is equality'
+    if (!pass) {
+      let reminderMessage = 'If it should pass with deep equality, replace "toBe" with "toStrictEqual"\n\n'
+      const toStrictEqualPass = jestEquals(
+        actual,
+        expected,
+        [
+          iterableEquality,
+          typeEquality,
+          sparseArrayEquality,
+          arrayBufferEquality,
+        ],
+        true,
+      )
+
+      if (toStrictEqualPass) {
+        toBeMessage = `${reminderMessage}expected #{this} to be [serializes to the same string] // Object.is equality`
+      }
+      else {
+        const toEqualPass = jestEquals(
+          actual,
+          expected,
+          [iterableEquality],
+        )
+
+        if (toEqualPass) {
+          reminderMessage = 'If it should pass with deep equality, replace "toBe" with "toEqual"\n\n'
+
+          toBeMessage = `${reminderMessage}expected #{this} to be [serializes to the same string] // Object.is equality`
```

Comment:
`expected to be` should be at the top. this is error message.

`expected/received` block is rendered independently

Reply:
Solved in commit [9cc61](https://github.com/vitest-dev/vitest/pull/1383/commits/9cc61ea2bdba3ed7cab1a93bb9020ed98e8356c9).